app/views/comments/_form.html.erb

```diff
@@ -54,7 +54,7 @@
     </div>
   </div>
 
-  <div class="well col-md-11" id="preview" style="background:white;display: none">
```

Comment:
This is preview tab and not comment form.

Reply:
Which means you are targetting wrong div.

Reply:
oh no.. thanks for pointing that out!